### ISSUE TITLE
No longer available in chrome store

### ISSUE THREAD

**Issue by Mar01** [yggdrasil-group/Hide-Most-Visited-Pages-Reloaded#9]
It can't be found in the chrome web store via search, and the link provided in the readme returns 404 not found.

**Comment by emkae** [comment]
That is correct. I haven't updated the readme, yet. I pulled the app from the webstore. Too much hatred and false accusations. Wasn't worth keeping it up anymore.

**Comment by HumbleKagu** [comment]
I really liked using this extension. It was the only one that seemed to work for me. Now that its not on the store it doesn't work for me anymore. There is no chance of you bringing it back at all?

**Comment by Mar01** [comment]
>That is correct. I haven't updated the readme, yet. I pulled the app from the webstore. Too much hatred and false accusations. Wasn't worth keeping it up anymore.

Sorry to hear that. :(

**Comment by emkae** [comment]
Probably I'm going to re-publish it as a closed "beta", so some of the more dedicated and friendly users and me can still use it as they were before. Next release will feature custom CSS and JavaScript so you can add JavaScript Widgets like weather to your newtab page if you like.

**Comment by HumbleKagu** [comment]
That is great to hear!

**Comment by emkae** [comment]
Current work in progress, but works pretty for me.

![screenshot from 2017-07-26 19-18-15](https://user-images.githubusercontent.com/1384938/28634378-4ad445fe-7237-11e7-80ca-5903cf35f814.png)
![screenshot from 2017-07-26 19-18-29](https://user-images.githubusercontent.com/1384938/28634379-4ae34108-7237-11e7-81e8-1efe7007ee4c.png)

**Comment by MrAndersonX** [comment]
Well, it seems to work when opening new tabs and stuff, but when opening Google Chrome for the first time though, it doesn't work. If you reload the page or open a new tab it does...Is this a new bug ? It didn't used to do this before....I have : Also Hide Google Logo, Also Hide Google Searchbar checked, so it's pretty easy to spot. It just doesn't want to work when I open Chrome, even if I have selected the "Open the new tab page" in Chrome options as home.

**Comment by emkae** [comment]
This seems to be a newly introduced bug. I'm afraid, but I'm away from the
computer for the next 3 full days and can't fix it until Monday. Hope this
works for you.


Best

Marco

**Comment by MrAndersonX** [comment]
Okay, thanks a bunch, just wanted to let you know :). It's sad that you had to remove it from the store...Hope it'll stay, becomes it's awesome ! One of those few addons we really need, but gets ignored by the enterprises and it's up to solo or small team of people to keep them up...I know it sucks and I get the feeling...Having something everyone wants, but few people wanna support or help achieve it.

**Comment by josh18** [comment]
@emkae sorry for the negative reviews, I really appreciate the extension especially the fact that it is open source. Thanks for your time and effort!

**Comment by xlollomanx** [comment]
Sorry to awake this old thread but I hope you continue to develop this addon. It's very sad to hear many people despise you work, it's an awesome addon and it works until today. On chrome 61 released today it not work anymore :(

**Comment by txtad** [comment]
HOW could anyone hate on this app? 

The only thing I hate is that it stopped working for me today, and I just found this thread and discovered why. I sure hope you republish it ASAP.

**Comment by lunix4all** [comment]
Until this comes back to the Chrome Web Store, here's how you can hide you most visited pages (If you're on Linux)

Chromium:

sed -i 's/id="most-visited"/id="most-visitod"/g' /usr/lib/chromium/resources.pak

Chrome

sudo /usr/bin/sed -i 's/id="most-visited"/id="most-visitod"/g' /opt/google/chrome/resources.pak

**Comment by xlollomanx** [comment]
@lunix4all Did u know any solution or workaround on windows too?

**Comment by Gypsy46** [comment]
One of the most reliable extensions I have come across. I have it on 3 computers and was about to install it on my 10 year old's new laptop so he can see his Batman theme without all the crap. Really sad that something so good can no longer be accessed. I tried installing it using the zip file, but Chrome is telling me that the manifest file is missing. I don't know maybe rename it and make it available again and just ignore the idiots out there!

**Comment by mhxion** [comment]
I never liked the most visited tiles on chrome, this extension was the only **better**(other store themes/blank tab totally suck) way to get rid of those. I've been using it for long, now it's gone and still no working version release. The one [in here](https://www.crx4chrome.com/crx/48650/) doesn't seem to be working anymore, not even the custom styling. Hope @walialu pulls the trigger soon.

**Comment by katenie** [comment]
One of my fav extensions of Chrome and sorry for the negative reviews. Thanks for your effort!

**Comment by Stasis-Windwalk** [comment]
The best extension I have ever used.
Pretty please with sugar on top please please please bring this back! T-T The google junk on my tabs is driving me insane! Make it go away! @walialu save us!
[Made an account just to beg you to bring it back because I loved it that much]

**Comment by Qyour** [comment]
Sad it doesn't work anymore...
I hope walialu will make this extension great again.

**Comment by MrAndersonX** [comment]
Walialu, 
Can you please send me the extention/addon again ? Pretty please ? I removed it today by accident and can't find it anywhere (yeah I know u removed it in Chrome). It worked fine for me, but now it's gone only cause I got stupid :|. Can you please help :P ?

**Comment by RipTornSideways** [comment]
Any way I could get the code for this to use in something like Tampermonkey?

**Comment by Felina-Lain** [comment]
I tried downloading and installing it directly, I dragged the SRC folder in chrome, but it tells me it has an error and can't find background.js
I looked and the file is in the js folder, so not sure what I did wrong

shame, this was the best extension for me, I wanted it for a new pc and now I can't have it :c

**Comment by mhxion** [comment]
If anyone's interested in this, here's a simple script I just released that do the same thing and works on the current version. You can develop your own theme if you wish for. https://github.com/mhxion/new-new-tab-for-chrome

**Comment by walialu** [comment]
@mhxion your contribution is appreciated, but your extension does nothing like this one did before.

Your extension [simply overrides the default newtab page](https://github.com/mhxion/new-new-tab-for-chrome/blob/master/manifest.json#L19). This extension modified the default newtab page.

**Comment by mhxion** [comment]
@walialu I see. I totally forgot about the other country domains, it only works for the US users then. I think there's a way to solve this though, a script that identifies the user's IP address and then redirects to his particular Google domain tab, I mean for example  `.co.jp/_chrome/newtab` What do you think about that? Thanks for pointing out the issue. 

Well, a good news is Google is [pulling out its country-based domains](https://www.blog.google/products/search/making-search-results-more-local-and-relevant/) soon and therefore everyone from every country should experience the same.

**Comment by walialu** [comment]
@mhxion it's not the most elegant solution, but it worked quite well, so you could simply use this full list of domains here: https://github.com/superevilmegaco/Hide-Most-Visited-Pages-Reloaded/blob/master/src/manifest.json#L17

**Comment by mhxion** [comment]
I've already checked that out. Thanks.

**Comment by Carbon-14th** [comment]
Hi walialu I am newbie and i have some irrelevant question
1. i was trying your build but when i tap the new tab icon the bookmark bar didn't appear. Is it a bug or something?
2. is that possible to use theme as background instead of using a single color as background, if yes, how?
i'm sorry to hear that some others didn't appreciate your work but you really did hell of a job

**Comment by walialu** [comment]
> i was trying your build but when i tap the new tab icon the bookmark bar didn't appear. Is it a bug or something?

Just press <kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>b</kbd> and it should appear and stay visible.

> is that possible to use theme as background instead of using a single color as background, if yes, how?

Yes, indeed this is possible.

Just upload your image to a free hosting like https://imgur.com or https://ctrlq.org/images/ and use the direct link to the image and add this (with your URL, instead of the example url) to your **Custom CSS**:

```css
body {
  background-image: url("https://i.imgur.com/jBMUI9H.png") no-repeat;
  background-size: cover;
}
```
